int16 planeID
float64 latitude
float64 longitude
float64 altitude
bool isAvoidanceManeuver
bool isNewQueue
---
string error
